### PR TITLE
Release notes for 0.25.5

### PR DESCRIPTION
**Pull request by rdodev**
@paolaperaza please 👀

### CODE REVIEW

**Comment by vishwas-astro** [comment]
@rdodev - Even though both features (Air gapped and hard delete) are not part of our prod cloud deployment, we should still create a release note entry there since we will push 0.25.5 to our cloud.

**Comment by rdodev** [comment]
> @rdodev - Even though both features (Air gapped and hard delete) are not part of our prod cloud deployment, we should still create a release note entry there since we will push 0.25.5 to our cloud.

Addressed PTAL!

**Review by paolaperaza — APPROVED** [review]
@rdodev I just made a few changes here!

- Removed Enterprise-only changes from Cloud release notes
- Fixed a few typos/cleaned up copy
- Expanded on hard deletion

I think this LGTM. The only addition I'd make is potentially adding more specific instructions for how to enable the hard-delete feature. Are those documented anywhere? (e.g. what's the name of the value in a user's config.yaml)

**Comment by rdodev** [comment]
@paolaperaza great. Thanks for changes/input. Will merge or would you rather wait until we're closer to release?

**Comment by paolaperaza** [comment]
@rdodev You got it! Feel free to make any additional changes on top of these in case I/we missed anything.

I'm not looped into the 0.25.5 roll-out, so I'll let you or @vishwas-astro merge this PR as soon as you're ready. We typically wait to merge until the external helm repo has been updated, but will defer to you.

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 1]
```diff
@@ -12,9 +12,28 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
...
+
+### Ability to Hard Delete Airflow Deployments
+
+Astronomer v0.25.5 introduces a platform-level setting that enables the ability for Deployment Admins to hard delete Airflow Deployments from Astronomer. Once that setting is enabled, users can choose to hard delete a Deployment in the **Settings** tab of the Astronomer UI or by running `$ astro deployment delete --hard` via the Astronomer CLI.
+
+This action permanently deletes all data associated with a Deployment that cannot be recovered, including the database and underlying Kubernetes resources. If your team has enabled [custom release names](https://www.astronomer.io/docs/enterprise/v0.25/deploy/configure-deployment#customize-release-names), hard deleting a Deployment unlocks its release name for reuse by another Deployment.
+
+### Airgapped Installation
+
+For customers interested in running Astronomer in an airgapped environment with no access to the internet, we've added the ability to do so. To learn more, contact [Astronomer Support](https://support.astronomer.io).
```
Comment (jwitz) [inline enterprise/v0.25/10_resources/01_release-notes.md:27]:
```suggestion
Organizations can now install Astronomer in an airgapped environment with no access to the internet. To learn more about this installation process, contact [Astronomer Support](https://support.astronomer.io).
```

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 2]
```diff
@@ -12,9 +12,28 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
 
 We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
 
+## v0.25.5
+
+Release Date: July 29, 2021
+
+### Ability to Hard Delete Airflow Deployments
+
+Astronomer v0.25.5 introduces a platform-level setting that enables the ability for Deployment Admins to hard delete Airflow Deployments from Astronomer. Once that setting is enabled, users can choose to hard delete a Deployment in the **Settings** tab of the Astronomer UI or by running `$ astro deployment delete --hard` via the Astronomer CLI.
```
Comment (jwitz) [inline enterprise/v0.25/10_resources/01_release-notes.md:21]:
```suggestion
Astronomer v0.25.5 introduces the ability for Deployment Admins to hard delete Airflow Deployments from Astronomer. This feature must first be enabled in your platform configuration before users can access it. 

Once the feature is enabled, users can hard delete a Deployment in the **Settings** tab of the Astronomer UI or by running `astro deployment delete --hard` via the Astronomer CLI.
```

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 3]
```diff
@@ -12,9 +12,28 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
...
+
+This action permanently deletes all data associated with a Deployment that cannot be recovered, including the database and underlying Kubernetes resources. If your team has enabled [custom release names](https://www.astronomer.io/docs/enterprise/v0.25/deploy/configure-deployment#customize-release-names), hard deleting a Deployment unlocks its release name for reuse by another Deployment.
+
```
Comment (jwitz) [inline enterprise/v0.25/10_resources/01_release-notes.md:24]:
```suggestion

To enable this feature, set `deployments.hardDeleteDeployment = true` in your `config.yaml` file and push the changes to your platform as described in [Apply a Config Change](https://www.astronomer.io/docs/enterprise/v0.25/manage-astronomer/apply-platform-config).

```

**Review by jwitz — COMMENTED** [review]
Adding some edits here. @rdodev can you check to see if the YAML value I suggested is formatted correctly? Do we need to add `houston-api` or something similar before `deployments`?

**Comment by rdodev** [comment]
@jwitz @paolaperaza I wonder if we should to get into the weeds for release notes? In fact, I would advocate for removing the flag and all of that detail. Just my 2c, but release notes should be a "bill of materials", not really documentation or implementation details.

**Comment by jwitz** [comment]
@rdodev In general I agree, but oftentimes we find ourselves using release notes to add information that doesn't quite have a home in any of our existing docs. I imagine this will be much less of an issue once we add some key docs and tighten our process up around releasing and documenting changes.

Thankfully for now, I think I found a spot in our docs where we can add this info. Checkout [this PR ](https://github.com/astronomer/docs/pull/418) and let me know what you think. If that looks good, I'll greatly simplify the release note here and link out to the doc instead.

**Comment by rdodev** [comment]
@jwitz that looks good and would definitely tidy up all the gritty details from release notes.

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 4]
```diff
@@ -12,9 +12,26 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
...
+Release Date: July 29, 2021
+
+### Ability to Hard Delete Airflow Deployments
+
+Astronomer v0.25.5 introduces a platform-level setting that enables the ability for Deployment Admins to hard delete Airflow Deployments from Astronomer. For more information about this feature, read [Delete a Deployment](https://www.astronomer.io/docs/enterprise/v0.25/deploy/configure-deployment#delete-a-deployment.
+
+### Airgapped Installation
+
+For customers interested in running Astronomer in an airgapped environment with no access to the internet, we've added the ability to do so. To learn more, contact [Astronomer Support](https://support.astronomer.io).
```
Comment (jwitz) [inline enterprise/v0.25/10_resources/01_release-notes.md:25]:
```suggestion
Organizations can now install Astronomer in an airgapped environment with no access to the internet. To learn more, contact [Astronomer Support](https://support.astronomer.io).
```

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 5]
```diff
@@ -12,9 +12,26 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
 
 We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
 
+## v0.25.5
+
+Release Date: July 29, 2021
+
+### Ability to Hard Delete Airflow Deployments
+
+Astronomer v0.25.5 introduces a platform-level setting that enables the ability for Deployment Admins to hard delete Airflow Deployments from Astronomer. For more information about this feature, read [Delete a Deployment](https://www.astronomer.io/docs/enterprise/v0.25/deploy/configure-deployment#delete-a-deployment).
```
Comment (jwitz) [inline enterprise/v0.25/10_resources/01_release-notes.md:21]:
```suggestion
Astronomer v0.25.5 introduces a platform-level setting that gives Deployment Admins the ability to hard delete Airflow Deployments. For more information about this feature, read [Delete a Deployment](https://www.astronomer.io/docs/enterprise/v0.25/deploy/configure-deployment#delete-a-deployment).
```

**Pull request merged**